.eslintrc.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  plugins: ['prettier'],
+  parser: '@typescript-eslint/parser',
+  extends: [
+    'plugin:react/recommended',
+    'plugin:@typescript-eslint/recommended',
+    'prettier/@typescript-eslint',
+    'plugin:prettier/recommended',
+  ],
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+  settings: {
+    react: {
+      version: 'detect',
+    },
+  },
+  rules: {
+    'prettier/prettier': 'error',
+  },
+};
```

Comment:
I don't know if we prefer `.js` or `.json`?  It's possible to put comments in the `.js` file without the `jsonc` workaround.